envs/example/defaults-2.0.yml

```diff
@@ -638,3 +638,11 @@ serverspec:
 
 inspec:
   enabled: False
+
+#openstack-ansible-security overrides
+security_enable_chrony: false
+security_ssh_permit_root_login: without-password
```

Comment:
I agree, this should be deny, but I was setting it to what is currently set in Ubuntu. I wanted a confirmation that this would not cause any issues during maintenance.